source/_patterns/01-atoms/dropdown/demo/dropdowns.twig

```diff
@@ -0,0 +1,201 @@
+<div class="container">
+  {% embed '@atoms/grid/_grid-1-up.twig' with { container: true } %}
+    {% block column_1 %}
+
+      <h1>Dropdowns</h1>
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_text: dropdowns_helper,
+        card_background: 'secondary',
+        text_color: 'light',
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/components/dropdowns/',
+        }
+      } %}
+
+      <div class="mb-3">
+        <h1>Basic Dropdown</h1>
+        {% include '@atoms/dropdown/_dropdown.twig' with {
+          dropdown_color: 'secondary',
+          dropdown_text: 'Dropdown button',
+          button_id: 'dropdownMenuButton',
+        } %}
+      </div>
+      <div class="mb-3">
+        <h1>&lta&gt element dropdown</h1>
+        {% include '@atoms/dropdown/_dropdown.twig' with {
+          dropdown_color: 'secondary',
+          dropdown_text: 'Dropdown link',
+          button_id: 'dropdownMenuLink',
+          button_element: 'a',
+          unique_attributes: 'role="button" id="dropdownMenuLink"',
+          button_link: '#',
+        } %}
+      </div>
+
+      <div class="mb-3">
+        <h1>Colors</h1>
+        {% for name, rgb in scssColors %}
+          {% include '@atoms/dropdown/_dropdown.twig' with {
+            dropdown_button: 'true',
+            dropdown_color: name,
+            dropdown_text: name|capitalize,
+          }%}
```

Comment:
Hyped Pedantic Overkill Comment: should we decide on a consistent style here between `}%}` and `} %}`?

Reply:
That was just a mess up on my part. Fixing